exercises/darts/canonical-data.json

```diff
@@ -77,6 +77,15 @@
       },
       "expected": 10
     },
+    {
+      "property": "score",
+      "description": "A dart lands right in the center of the dart board is scored in the inner circle",
```

Comment:
I like this test case. Maybe this should actually be the very first test case, as it requires the least amount of code to get right.

Reply:
Yeah, I thought about putting it first, but figured that the test case where you're returning 0 might be better? It seemed like the test suite is vaguely ordered to work from the outside inwards.

Reply:
Ah right. That makes sense.